manageability-docker/build-az-container.sh

```diff
@@ -1,4 +1,8 @@
 #!/bin/bash
```

Comment:
I think we should use the full name for Azure and Thingsboard.  It's not going to be clear to everyone and since we are open source I think it should be clear what the scripts are targetting.